roles/openshift_logging_curator/tasks/main.yaml

```diff
@@ -56,6 +56,14 @@
     dest: "{{ tempdir }}/curator.yml"
   changed_when: no
 
+- copy:
+    src: actions.yaml
```

Comment:
Seems this is a higly configurable part of curator for which we would like users to be able to define at deployment time instead of having to edit a configmap.  I would add a variable here that  defines the path to an action file that we should use and fallback to our default if it is undefined.  This will allow ansible to be the 'single source of truth' and keep them from having to rebuild the action file upon every deployment.

Reply:
I'm on the fence about that. Currently we document to edit the configmap after deployment has happened, though we recommended scaling down and then up to then kick it off again -- do we have a similar story for cron jobs?

We should be safe from them rebuilding it after each deployment since we can leverage the patching logic already in place (https://github.com/openshift/openshift-ansible/pull/7047/files#diff-f4c6ebdf7da2e4a833073d1f89e08221R84)